mgmtsystem_action/views/mgmtsystem_action_stage.xml

```diff
@@ -1,28 +1,95 @@
 <odoo>
-  <data>
 
-    <record model="ir.ui.view" id="mgmtsystem_action_stage_tree_view">
-      <field name="name">Management System action Stage Tree</field>
-      <field name="model">mgmtsystem.action.stage</field>
-      <field name="arch" type="xml">
-        <tree string="Stages">
-          <field name="name"/>
-          <field name="sequence" widget="handle" />
-          <field name="is_starting"/>
-          <field name="is_ending"/>
-        </tree>
-      </field>
-    </record>
+        <record id="mgmtsystem_action_stage_tree_view" model="ir.ui.view">
+            <field name="name">mgmtsystem.action.stage.tree</field>
+            <field name="model">mgmtsystem.action.stage</field>
+            <field name="arch" type="xml">
+                <tree string="Action Stage">
+                    <field name="name"/>
+                    <field name="sequence" widget="handle" groups="base.group_no_one"/>
```

Comment:
@grindtildeath 
thats also not needed, since ill be splitting migration code and new features, this will be left out as well. 

Thanks again for the good review...
Also thanks to @sbidoul for spotting the linting errors and the test failures.
As for the failures im still working on it.